experiment/nursery/create_bazel_cache_rcs.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+# Copyright 2018 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# THIS IS TEMPORARY (!)
```

Comment:
This needs to be stuck to. If this grows logic or goes into use on actual jobs it needs to be rewritten. K8s doesn't need more piles of bash :+)

Reply:
you're only allowed to check in new lines of bash if you delete an equal or greater number of lines of bash somewhere else in the same PR.

Reply:
:smile: I could probably delete `hack/bazel-test-affected.sh` 🤔 
There seems to be a bit of deletable bash in test-infra actually..

Reply:
^^ that bash is dead and is `--affected` flag equivalent in bazel scenario